internal/cmd/docker/push.go

```diff
@@ -129,6 +129,7 @@ func logPushProgress(reader io.ReadCloser) error {
 
 			// Create a progress spinner for statuses that don't have progress details, like 'Preparing'.
 			if msg.Progress == nil || msg.Progress.Total == 0 {
+				//nolint:govet
 				progress.Show(msg.Status)
```

Comment:
The linter is satisfied if you show explicitly that there are no args:
```suggestion
				progress.Show(msg.Status, nil)
```